lib/iphone-simulator.js

```diff
@@ -122,12 +129,19 @@ var iPhoneSimulator = (function () {
         }
         simulator.setSimulatedDevice(config);
 
-        if (options.stderr) {
-            config("setSimulatedApplicationStdErrPath", $(options.stderr));
-        }
-
-        if (options.stdout) {
-            config("setSimulatedApplicationStdOutPath", $(options.stdout));
+        if (options.logging) {
+            var logPath = path.join(path.dirname(appPath), path.basename(appPath, ".app") + ".stderr");
+            child_process.execSync(util.format("rm -f %s && mkfifo %s", logPath, logPath));
```

Comment:
execSync is available only on node 0.12 and io.js but we support node 0.10 as well.